integration_tests/spiaccesstokencontroller_test.go

```diff
@@ -75,6 +77,85 @@ var _ = Describe("Create without token data", func() {
 	})
 })
 
+var _ = Describe("Status", func() {
+	var createdToken *api.SPIAccessToken
+	var origBaseUrl string
+	var origOAuthUrl string
+	var origUploadUrl string
+
+	triggerReconciliation := func() {
+		Eventually(func(g Gomega) {
+			// trigger the update of the token to force the reconciliation
+			currentToken := &api.SPIAccessToken{}
+			g.Expect(ITest.Client.Get(ITest.Context, client.ObjectKeyFromObject(createdToken), currentToken)).To(Succeed())
+			currentToken.Annotations = map[string]string{"random-anno": string(uuid.NewUUID())}
+			g.Expect(ITest.Client.Update(ITest.Context, currentToken)).To(Succeed())
+		}).Should(Succeed())
+	}
+
+	BeforeEach(func() {
+		ITest.TestServiceProvider.Reset()
+		origBaseUrl = ITest.OperatorConfiguration.BaseUrl
+		ITest.OperatorConfiguration.BaseUrl = "https://initial.base.url"
+		ITest.TestServiceProvider.GetOauthEndpointImpl = func() string {
+			return ITest.OperatorConfiguration.BaseUrl + "/test/oauth"
+		}
+
+		createdToken = &api.SPIAccessToken{
```

Comment:
Is that test stable (keeping in mind that tokens w/out linked bindings may eventually dissappear due to cleaner)

Reply:
The testsuite sets the TTL of bindings and tokens to 10s. So it imho doesn't make a difference. I would actually say that we should by default set the TTLs to a much larger value to lower the probability of the TTL intervention and only set it to short TTLs in the dedicated tests.

Reply:
I am talking about cleaner of `Awaiting...` state tokens w/out linked bindings. They're did not have TTL. It probably works only due to some 2s grace period now.

Reply:
I made the deletion grace period configurable and set it to 10s in the integration test setup.